src/utils/ImageCreator.cpp

```diff
@@ -274,34 +277,34 @@ class TOPPImageCreator :
         return ILLEGAL_PARAMETERS;
       }
     }
+    const double init = numeric_limits<double>::max();
+    double rt_min = -init, rt_max = init, mz_min = -init, mz_max = init;
+    bool filter_rt = parseRange_(getStringOption_("rt"), rt_min, rt_max);
+    if (rt_min > rt_max) swap(rt_min, rt_max);
```

Comment:
@hendrikweisser: make this part of `parseRange_` to fix min/max if user provides it incorrectly?